stanley/helpers.py

```diff
@@ -11,10 +11,12 @@ def request_feedback():
     sender = get_sender(members)
     receiver = get_receiver(members, sender)
 
-    # set sender, receiver pair in the storage
-    redis_storage.set(sender[0], receiver[0])
     message = 'Hey, tell me something nice about @{}'.format(receiver[1])
-    send_slack_message(channel='@{}'.format(sender[0]), message=message)
+    response = send_slack_message(channel='@{}'.format(sender[0]), message=message)
+
+    if response['ok']:
```

Comment:
What about we [put a warning](http://flask.pocoo.org/docs/dev/logging/#logging) or even [send a sentry error](https://docs.sentry.io/clients/python/integrations/flask/#usage) if it was not ok? It would be good to know how often this could fail and for what reasons.